src/main/java/org/jenkinsci/plugins/buildgraphview/BuildGraph.java

```diff
@@ -72,10 +72,12 @@ private void computeGraphFrom(BuildExecution b) throws ExecutionException, Inter
         for (DownStreamRunDeclarer declarer : DownStreamRunDeclarer.all()) {
             List<Run> runs = declarer.getDownStream(run);
             for (Run r : runs) {
-                BuildExecution next = getExecution(r);
-                graph.addVertex(next);
-                graph.addEdge(b, next, new Edge(b, next));
-                computeGraphFrom(next);
+                if (r !=null) {
```

Comment:
Seems wrong. A `DownStreamRunDeclarer` implementation should be responsible for not including any nulls in its return value. Find the faulty one and fix it.